test/test_multi_region_cubed_sphere.jl

```diff
@@ -129,13 +183,34 @@ create_ψ_test_data(grid, region) = create_test_data(grid, region; trailing_zero
 create_u_test_data(grid, region) = create_test_data(grid, region; trailing_zeros=2)
 create_v_test_data(grid, region) = create_test_data(grid, region; trailing_zeros=3)
 
+function create_test_data_in_lexicographic_order(grid, region; trailing_zeros=0)
```

Comment:
I don't know what "lexicographic order" is; perhaps a short docstring is in order?